conbench/api/commits.py

```diff
@@ -18,7 +20,12 @@ def get(self):
         tags:
           - Commits
         """
-        commits = Commit.all(order_by=Commit.timestamp.desc(), limit=500)
+        commit = f.request.args.get("commit")
+        if commit:
+            commit_list = commit.split(",")
+            commits = Commit.search([Commit.sha.in_(commit_list)])
+        else:
+            commits = Commit.all(order_by=Commit.timestamp.desc(), limit=500)
```

Comment:
```suggestion
        if sha_arg := f.request.args.get("sha"):
            shas = sha_arg.split(",")
            commits = Commit.search([Commit.sha.in_(shas)])
        else:
            commits = Commit.all(order_by=Commit.timestamp.desc(), limit=500)
```

Reply:
Oh this is quite nifty. I think we also do this in benchmark endpoints for consistency.